### PR TITLE
R

### PR DESCRIPTION
**Pull request by taneeshaa15**
Hi @codewarnab   ,
I hope you're doing well. I've addressed the issue #505  and created a pull request to resolve it.
Could you please review the PR and let me know if any adjustments are needed?

Best regards,
Tanisha Garg

### CODE REVIEW

File: src/App.css
Comment (codewarnab) [inline src/App.css:1]: you just chanaged the app.css fonts path how its #505 ? 
also this is an vite  app public folder is eserved as root path why did you chanaged it ?

**Review by codewarnab — CHANGES_REQUESTED** [review]

File: src/App.css
Comment (taneeshaa15) [reply src/App.css:1]: I just tried to make it optimizable for mobile screens.

**Comment by taneeshaa15** [comment]
@codewarnab Tell me the adjustments needed now.

**Comment by codewarnab** [comment]
add screenshot of team page

**Comment by taneeshaa15** [comment]
@codewarnab Will complete it as soon as possible.

**Comment by taneeshaa15** [comment]
@codewarnab Please review.

**Comment by codewarnab** [comment]
> add screenshot of team page

.

**Comment by taneeshaa15** [comment]
@codewarnab Here is the screenshot of team page after updation:

Screenshot from 2024-07-02 20-47-07

**Pull request merged**